packages/packages/metriport-sdk/src/medical/client/metriport.ts

```diff
@@ -0,0 +1,26 @@
+import axios, { AxiosInstance } from "axios";
+
+export class MetriportMedicalApi {
+  private api: AxiosInstance;
+
+  /**
+   * Creates a new instance of the Metriport API client.
+   *
+   * @param {string} apiKey - Your Metriport API key.
+   */
+  constructor(apiKey: string, baseURL = "https://api.metriport.com") {
```

Comment:
base URL can be `"https://api.metriport.com/medical"`

Reply:
Since we're defining the API for medical now, what about adding `v1` there? Like `https://api.metriport.com/medical/v1`